From 0288a613bba0dfac84281fd6afc9e7a5008fdef5 Mon Sep 17 00:00:00 2001
From: b07117 <b07117@freescale.com>
Date: Thu, 24 Nov 2011 00:43:44 +0800
Subject: [PATCH 214/498] ENGR00000000 fixing build error for mx5x ICS

Copybit HAL is removed in ICS, disable mx5x copybit
Removing overlay header in mx5x hwcomposer

Signed-off-by: xianzhong <b07117@freescale.com>
---
 mx5x/hwcomposer/hwcomposer.cpp | 2 +-
 mx5x/libcopybit/Android.mk     | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/mx5x/hwcomposer/hwcomposer.cpp b/mx5x/hwcomposer/hwcomposer.cpp
index d82802c..8cb6444 100755
--- a/mx5x/hwcomposer/hwcomposer.cpp
+++ b/mx5x/hwcomposer/hwcomposer.cpp
@@ -15,7 +15,7 @@
 /*Copyright 2009-2011 Freescale Semiconductor, Inc. All Rights Reserved.*/
 
 #include <hardware/hardware.h>
-#include <hardware/overlay.h>
+//#include <hardware/overlay.h>
 
 #include <fcntl.h>
 #include <errno.h>
diff --git a/mx5x/libcopybit/Android.mk b/mx5x/libcopybit/Android.mk
index 6f06443..d399ce0 100755
--- a/mx5x/libcopybit/Android.mk
+++ b/mx5x/libcopybit/Android.mk
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+ifeq (true, false)
 ifeq ($(HAVE_FSL_IMX_GPU),true)
 ifeq ($(BOARD_SOC_CLASS),IMX5X)
 
@@ -42,3 +43,4 @@ include $(BUILD_SHARED_LIBRARY)
 
 endif
 endif
+endif
-- 
1.8.0

